examplenuget/windows/examplenuget/examplenuget.vcxproj

```diff
@@ -176,6 +166,7 @@
     </Page>
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="Microsoft.Windows.CppWinRT" Version="2.0.211028.7" />
     <ProjectReference Include="..\..\..\package\windows\ReactNativeXaml\ReactNativeXaml.vcxproj">
```

Comment:
Does autolinking work if you remove this Project Reference and let it get generated? the path may be "different" (in node_modules) but that should just be a link back to the correct folder anyway.

Reply:
Still hits the same issue, it keeps trying to change the pathing of react-native-xaml to the one in node-modules (ie from "$(ProjectDir)..\..\..**\package**\windows\ReactNativeXaml\ReactNativeXaml.vcxproj" to $(ProjectDir)..\..\..**\node_modules**\react-native-xaml\windows\ReactNativeXaml\ReactNativeXaml.vcxproj")